src/prisma/schema.prisma

```diff
@@ -3507,3 +3508,26 @@ model ScoutProjectDeployer {
   @@index([projectId])
   @@index([address])
 }
+
+model PartnerReward {
```

Comment:
rename to PartnerRewardPayout

and move contract-related fields to PartnerRewardPayoutContract